license-maven-plugin-git/src/main/java/com/mycila/maven/plugin/license/git/GitLookup.java

```diff
@@ -95,14 +93,15 @@ public GitLookup(File anyFile, DateSource dateSource, TimeZone timeZone, int che
         throw new IllegalStateException("Unexpected " + DateSource.class.getName() + " " + dateSource);
     }
     this.checkCommitsCount = checkCommitsCount;
+    this.commitsToIgnore = commitsToIgnore;
   }
 
   /**
    * Returns the year of the last change of the given {@code file} based on the history of the present git branch. The
    * year is taken either from the committer date or from the author identity depending on how {@link #dateSource} was
    * initialized.
    * <p>
-   * See also the note on time zones in {@link #GitLookup(File, DateSource, TimeZone, int)}.
+   * See also the note on time zones in {@link #GitLookup(File, DateSource, TimeZone, int, Set)}.
```

Comment:
This doesn't need to change if you use an overload. Also, interestingly, there is no note on time zones. 😁